src/web_app/js/call.js

```diff
@@ -419,23 +419,45 @@ Call.prototype.maybeCreatePcClient_ = function() {
   if (this.pcClient_) {
     return;
```

Comment:
Does it get called multiple times in a single call? 

I can understand the case where B leaves and rejoins, A uses the same PC, but it's not clear to me why this would be hit multiple times in a single call.